scheduling/gang-scheduling.md

```diff
@@ -0,0 +1,225 @@
+---
+title: gang-scheduling-in-kubernetes
+authors:
+  - "@ingvagabund"
+reviewers:
+  - "@damemi"
+approvers:
+  - "@soltysh"
+creation-date: 2020-02-27
+last-updated: 2020-02-27
+status: implementable
```

Comment:
Let's make it provisional for now, we don't plan to implement it, at least not yet.